simpleflow/swf/constants.py

```diff
@@ -12,3 +12,5 @@
     MAX_DECISIONS = 100
     MAX_OPEN_ACTIVITY_COUNT = int(os.getenv("SWF_MAX_OPEN_ACTIVITY_COUNT", 1000))
     MAX_REQUEST_SIZE = 1000 * 1000  # bytes
+
+TRACEBACK_SIZE = None  # Unlimited (well, up to 32K)
```

Comment:
Useless now with jumbo fields. I'd remove that and rely on standard truncation in `swf.format` module directly?